### PR TITLE
Enabling headless deployment in make file

### PR DESCRIPTION
**Pull request by estohlmann**
Enabling headless deployment in makefile. You can do this by running `make deploy HEADLESS=true`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -241,7 +242,13 @@ buildEcsDeployer:
 
 ## Deploy all infrastructure
 deploy: dockerCheck dockerLogin cleanMisc modelCheck buildEcsDeployer
-ifdef PROFILE
+ifneq (,$(findstring true, $(HEADLESS)))
+	if [ -z ${PROFILE} ]; then \
```
Comment (dustins) [inline Makefile:246]: Is this indentation actually off or just how github is showing it?

File: Makefile [hunk 2]
```diff
@@ -241,7 +242,13 @@ buildEcsDeployer:
...
+        npx cdk deploy ${STACK} -c ${ENV}='$(shell echo '${${ENV}}')'; \
+    else \
+        npx cdk deploy ${STACK} --profile ${PROFILE} -c ${ENV}='$(shell echo '${${ENV}}')'; \
```
Comment (bedanley) [inline Makefile:249]: We will want `cdk deploy --require-approval never` added so the deployer doesn't have to confirm stack changes.

File: Makefile [hunk 3]
```diff
@@ -241,7 +242,13 @@ buildEcsDeployer:
...
+    fi;
+else ifdef PROFILE
 	@printf "\n \
```
Comment (bedanley) [inline Makefile:252]: We should still print out the app Infrastructure properties regardless of headless mode

File: Makefile (hunk 1 above)
Comment (estohlmann) [reply Makefile:246]: makefile syntax can be quirky
- if you want a line of code to be interpreted as make code it must only be indented with spaces.
- if you want a line of code to be interpreted as bash code it must only be indented with tabs

**Review by bedanley — APPROVED** [review]

**Pull request merged**